config/generic-mks-sgenl-v2.cfg

```diff
@@ -0,0 +1,171 @@
+[printer]
+kinematics: cartesian
+max_velocity: 500
+max_accel: 3000
+max_z_velocity: 25
+max_z_accel: 30
+
+
+[mcu]
+    serial:/dev/serial/by-id/usb-Klipper_lpc1769_0D80000EA29869AFD0E63F5EC72000F5-if00
+
+##--------------------------------------------------------------------
+#  Stepper Settings
+##--------------------------------------------------------------------
+
+[stepper_x]
+step_pin: P2.2
+dir_pin: P2.3
+enable_pin: !P2.1
+rotation_distance: 40
+microsteps: 16
+full_steps_per_rotation: 200    #200 for 1.8 degree, 400 for 0.9 degree
+endstop_pin: ^P1.29
+position_endstop: -10
+position_min: -10
+position_max: 263
+
+
+[tmc2209 stepper_x]
+uart_pin: P1.1
+run_current: 0.7
+sense_resistor: 0.110
+##--------------------------------------------------------------------
+## Y Stepper
+##--------------------------------------------------------------------
+[stepper_y]
+step_pin: P0.19
+dir_pin: P0.20
+enable_pin: !P2.8
+rotation_distance: 40
+microsteps: 16
+full_steps_per_rotation: 200    #200 for 1.8 degree, 400 for 0.9 degree
+endstop_pin: ^P1.27
+position_endstop: -20
+position_min: -20
+position_max: 220
+
+## Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_y]
+uart_pin: P1.8
+run_current: 0.7
+sense_resistor: 0.110
+
+##--------------------------------------------------------------------
+#Z Stepper Settings
+##--------------------------------------------------------------------
+
+## Z0 Stepper
+[stepper_z]
+step_pin: P0.22
+dir_pin: P2.11
+enable_pin: !P0.21
+rotation_distance: 8
+full_steps_per_rotation: 200    #200 for 1.8 degree, 400 for 0.9 degree
+microsteps: 16
+endstop_pin: ^!P1.28
+position_endstop: 0
+position_min: -1.3
+position_max: 200
+
+
+## Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z]
+uart_pin: P1.10
+run_current: 0.7
+sense_resistor: 0.110
+
+## Z1 Stepper
+[stepper_z1]
+step_pin: P1.9
+dir_pin: P1.14
+enable_pin: !P0.10
+rotation_distance: 8
+full_steps_per_rotation: 200    #200 for 1.8 degree, 400 for 0.9 degree
+microsteps: 16
+
+[tmc2209 stepper_z1]
+uart_pin: P1.17
+run_current: 0.1
+sense_resistor: 0.110
+
+#####################################################################
+#Extruder
+#####################################################################
+[extruder]
+step_pin: P2.13
+dir_pin: !P0.11
+enable_pin: !P2.12
```

Comment:
It would be preferable to provide a commented blocks for E1 as well, in order to have a comprehensive cfg. See https://github.com/Klipper3d/klipper/blob/master/config/generic-bigtreetech-skr-v1.4.cfg as an example